samples/ask-akka-agent/src/main/java/akka/ask/indexer/application/RagIndexingWorkflow.java

```diff
@@ -102,7 +102,7 @@ public RagIndexingWorkflow(MongoClient mongoClient) {
         .createIndex(true)
         .build();
 
-    this.splitter = new DocumentByCharacterSplitter(500, 50, OpenAiUtils.buildTokenizer()); // <1>
+    this.splitter = new DocumentByCharacterSplitter(500, 50); // <1>
```

Comment:
wonder if this is the model embedding that we talked about? doesn't seem to be needed any more. OpenAiTokenizer doesn't exist